services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java

```diff
@@ -255,6 +257,14 @@ public RulesSyncResult synchronizeSwitchRules(SwitchId switchId) {
                 new HttpEntity(buildHeadersWithCorrelationId()), RulesSyncResult.class, switchId).getBody();
     }
 
+    @Override
+    public SwitchSyncResult synchronizeSwitch(SwitchId switchId, boolean removeExcess) {
```

Comment:
shouldn't there be separate flags for excess rules and excess meters?

Reply:
For now, API doesn't support separate flags for rules and meters. So we need to ask here #2189 about this. I will do it.